### PR TITLE
18 fix root component UI

### PR DESCRIPTION
**Pull request by roblumination**
1. Fixed height of root component. 
2. Moved header from app.module to core module
3. Edited header
> Added navigation menu
> Added adaptive styles

### CODE REVIEW

**Review by Lingsbay — APPROVED** [review]
Good.

File: src/app/core/components/header/header.component.scss [hunk 1]
```diff
@@ -1,7 +1,67 @@
-mat-toolbar {
-	justify-content: end;
-}
+@import '../../../../styles/variable.scss';
```
Comment (dennissimmo) [inline src/app/core/components/header/header.component.scss:1]: You can use @import 'variables.scss'

**Review by dennissimmo — APPROVED** [review]
All is good

**Review by AlexeySapr — APPROVED** [review]
All good

**Pull request merged**